integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/RefreshCarbonTableCommand.scala

```diff
@@ -83,6 +84,23 @@ case class RefreshCarbonTableCommand(
       if (FileFactory.isFileExist(schemaFilePath)) {
         // read TableInfo
         val tableInfo = SchemaReader.getTableInfo(identifier)
+        val tableProperties = tableInfo.getFactTable.getTableProperties
```

Comment:
Please verify direct upgrade scenarios in this case and handle if it doesnt work. Cam make changes in Refresh table to support this only for spatial table and update the document as well

Reply:
Verified for direct upgrade scenario and update document.